qemu/tests/usb_block_dev_hotplug.py

```diff
@@ -0,0 +1,104 @@
+import logging
+
+from virttest.qemu_devices import qdevices
+from virttest import data_dir
+from virttest import storage
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Test hotplug of usb block devices.
+
+    1) Boot up guest with usb block device
+    2) verify the usb block device
+    2) Unplug the usb device and verify
+    4) Hotplug block device and verify
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    def _verify_usb_device_in_monitor(device_type):
+        logging.info("Check usb device information in monitor")
+        output = str(vm.monitor.info("usb"))
+        if params["product"] not in output:
+            test.fail("Could not find %s" % device_type)
+
+    def _verify_usb_device_in_guest(session, device_serial):
+        logging.info("Check usb device information in guest")
+
+        # both Linux and Windows
+        def _check_usb_info():
+            output = session.cmd_output(params["chk_usb_info_cmd"],
+                                        float(params["cmd_timeout"]))
+            return (device_serial in output)
+        res = utils_misc.wait_for(_check_usb_info,
+                                  float(params["cmd_timeout"]),
+                                  text="Wait for getting usb device info")
```

Comment:
Because it is too fast, i am need to add `time.sleep(5)` before check_usb_info.
But I don't want to hard code that, so it will have tries